tests/integration/Statistics/Track/WooCommerceOrdersTest.php

```diff
@@ -0,0 +1,216 @@
+    expect(count(StatisticsWooCommerceOrders::findMany()))->equals(0);
+  }
+
+  function testItDoesNotTrackPaymentWithZeroCost() {
+    $this->createClick($this->link, $this->subscriber);
+    $order_mock = $this->createOrderMock($this->subscriber->email, 0);
+    $woocommerce_orders = new WooCommerceOrders($this->createWooCommerceHelperMock($order_mock));
+    $woocommerce_orders->trackPaidOrder($order_mock->get_id());
+    expect(count(StatisticsWooCommerceOrders::findMany()))->equals(0);
+  }
+
+  function testItDoesNotTrackPaymentWhenClickNewerThanOrder() {
+    $this->createClick($this->link, $this->subscriber, 0);
+    $order_mock = $this->createOrderMock($this->subscriber->email);
+    $woocommerce_orders = new WooCommerceOrders($this->createWooCommerceHelperMock($order_mock));
+    $woocommerce_orders->trackPaidOrder($order_mock->get_id());
+    expect(count(StatisticsWooCommerceOrders::findMany()))->equals(0);
+  }
+
+  function _after() {
+    \ORM::raw_execute('TRUNCATE ' . Newsletter::$_table);
+    \ORM::raw_execute('TRUNCATE ' . Subscriber::$_table);
+    \ORM::raw_execute('TRUNCATE ' . SendingQueue::$_table);
+    \ORM::raw_execute('TRUNCATE ' . StatisticsClicks::$_table);
+    \ORM::raw_execute('TRUNCATE ' . StatisticsWooCommerceOrders::$_table);
+  }
+
+  private function createNewsletter() {
+    $newsletter = Newsletter::create();
+    $newsletter->type = Newsletter::TYPE_STANDARD;
+    return $newsletter->save();
+  }
+
+  private function createQueue(Newsletter $newsletter, Subscriber $subscriber) {
+    $queue = Sending::create();
+    $queue->newsletter_id = $newsletter->id;
+    $queue->setSubscribers([$subscriber->id]);
+    $queue->updateProcessedSubscribers([$subscriber->id]);
+    return $queue->save();
+  }
+
+  private function createSubscriber($email) {
+    $subscriber = Subscriber::create();
+    $subscriber->email = $email;
+    $subscriber->first_name = 'First';
+    $subscriber->last_name = 'Last';
+    return $subscriber->save();
+  }
+
+  private function createLink(Newsletter $newsletter, Sending $queue) {
+    $link = NewsletterLink::create();
+    $link->hash = 'hash';
+    $link->url = 'url';
+    $link->newsletter_id = $newsletter->id;
+    $link->queue_id = $queue->id;
+    return $link->save();
+  }
+
+  private function createClick(NewsletterLink $link, Subscriber $subscriber, $created_days_ago = 5) {
+    $click = StatisticsClicks::create();
+    $click->newsletter_id = $link->newsletter_id;
+    $click->subscriber_id = $subscriber->id;
+    $click->queue_id = $link->queue_id;
+    $click->link_id = $link->id;
+    $click->count = 1;
+
+    $timestamp = new DateTime("-$created_days_ago days");
+    $click->created_at = $timestamp->format('Y-m-d H:i:s');
+    $click->updated_at = $timestamp->format('Y-m-d H:i:s');
+    return $click->save();
+  }
+
+  private function createWooCommerceHelperMock(PHPUnit_Framework_MockObject_MockObject $order_mock) {
+    $mock = $this->createMock(Helper::class);
```

Comment:
I think such generic names like `Helper` should be imported with an alias. When I am looking at this method I need to scroll up to understand what kind of helper this is.